cvat/requirements/base.txt

```diff
@@ -45,9 +45,7 @@ diskcache==5.0.2
 boto3==1.17.61
 azure-storage-blob==12.13.0
 google-cloud-storage==1.42.0
-# --no-binary=datumaro: workaround for pip to install
-# opencv-headless instead of regular opencv, to actually run setup script
-datumaro==0.2.0 --no-binary=datumaro
+git+https://github.com/cvat-ai/datumaro
```

Comment:
This link leads to the develop branch. I don't think it is a good solution, better to specify a branch or commit.

Reply:
Fixed